app/MyApp/index.html

```diff
@@ -1526,7 +1526,7 @@ <h3 class="bigheader">
                     </tr>
                     <tr>
                         <td>
-                            <input type="text" value="<%=answer%>" name="singleLineAnswer" style="margin-bottom: -45px;">
+                            <input type="text" value="<%=answer%>" name="singleLineAnswer">
```

Comment:
these style has been removed. are they added somewhere else or are they not required at all?

Reply:
Setting up margin for input field attributes

Reply:
they are no required at all